package.json

```diff
@@ -56,10 +56,10 @@
     "joi": "^17.7.0",
     "js-yaml": "^4.1.0",
     "keymbinatorial": "^2.0.0",
-    "screwdriver-data-schema": "^23.5.1",
+    "screwdriver-data-schema": "file:../data-schema/screwdriver-data-schema-22.0.0.tgz",
     "screwdriver-notifications-email": "^3.0.1",
     "screwdriver-notifications-slack": "^5.0.0",
-    "screwdriver-workflow-parser": "^4.3.0",
+    "screwdriver-workflow-parser": "file:../workflow-parser/screwdriver-workflow-parser-4.0.0.tgz",
```

Comment:
TODO: Update version after https://github.com/screwdriver-cd/workflow-parser/pull/55 is merged